### PR TITLE
adding liaison expectations

### PR DESCRIPTION
**Pull request by parispittman**
https://github.com/kubernetes/steering/issues/228

### CODE REVIEW

File: liaisons.md [hunk 1]
```diff
@@ -59,5 +59,43 @@ of SIGs, WGs and UGs.
 | [Committee Code of Conduct](committee-code-of-conduct/README.md) | Tim Pepper (**[@tpepper](https://github.com/tpepper)**) |
 | [Committee Security Response](committee-security-response/README.md) | Stephen Augustus (**[@justaugustus](https://github.com/justaugustus)**) |
 <!-- BEGIN CUSTOM CONTENT -->
+## Expectations
+### Liaison will: 
+- alert leads the Annual Report process is starting, expectations, and
```
Comment (dims) [inline liaisons.md:64]: Capital `A' for alert?

File: liaisons.md [hunk 2]
```diff
@@ -59,5 +59,43 @@ of SIGs, WGs and UGs.
 | [Committee Code of Conduct](committee-code-of-conduct/README.md) | Tim Pepper (**[@tpepper](https://github.com/tpepper)**) |
 | [Committee Security Response](committee-security-response/README.md) | Stephen Augustus (**[@justaugustus](https://github.com/justaugustus)**) |
 <!-- BEGIN CUSTOM CONTENT -->
+## Expectations
+### Liaison will: 
+- alert leads the Annual Report process is starting, expectations, and 
+- check-ins with their groups at least once a quarter. This could mean a group DM
```
Comment (dims) [inline liaisons.md:65]: Capital `C` for check-in

File: liaisons.md [hunk 3]
```diff
@@ -59,5 +59,43 @@ of SIGs, WGs and UGs.
...
+with Chairs, a visit to a group meeting, or some other connection. Get a pulse 
+on health of the following:
+  - Contributor base
+    - Chairs and other leads: How are they?
+    - Contributors, reviewers, approvers: How are they?
+    - Newcomers: able to efficiently join and contribute?
+  - Operational Health
+    - Transparency of meetings and decision processes (eg: published meeting 
+    - agendas, minutes, and video recordings)
```
Comment (dims) [inline liaisons.md:74]: drop the `-` in front of agendas as that means a new bullet point

File: liaisons.md [hunk 4]
```diff
@@ -59,5 +59,43 @@ of SIGs, WGs and UGs.
...
+    - Communication best practices for shared status and decision making across 
+    - a WG and its sponsor SIG(s) and a SIG and its subprojects
```
Comment (dims) [inline liaisons.md:76]: same here (`-`)

File: liaisons.md [hunk 5]
```diff
@@ -59,5 +59,43 @@ of SIGs, WGs and UGs.
...
+    - Provide consultation on how to improve operational health
+  - Gap Analysis
+    - where is the group now and where do they want/need to be?
```
Comment (dims) [inline liaisons.md:79]: `W`

File: liaisons.md [hunk 6]
```diff
@@ -59,5 +59,43 @@ of SIGs, WGs and UGs.
...
+- Check that sustainability is a priority. 
+  - As contributors and maintainers come and go, what is the group doing to 
+ensure contributions are sustainable? What does the group need in order to keep 
+things going? Are there areas of concern?
+- Update Steering Committee keep the Steering Committee in the loop and keep 
+track of trackable (ie: not sensitive) work in public issues.
+- Connect leads to whatever they need, when they ask. (if possible!)
+
+### Liaisons should:
+- communicate with their groups on important business that happens at Steering
```
Comment (dims) [inline liaisons.md:91]: `C`

File: liaisons.md [hunk 7]
```diff
@@ -59,5 +59,43 @@ of SIGs, WGs and UGs.
...
+- Connect leads to whatever they need, when they ask. (if possible!)
+
+### Liaisons should:
+- communicate with their groups on important business that happens at Steering 
+level (via our meetings, etc) in order for Steering to achieve a pulse on the 
+work we are doing and acknowledge governance changes. 
+  - Example: if Steering is crafting a new policy, check in with the chairs to 
+see how it would fare with their groups.
+
+be an advisor for a group’s requests for funding from CNCF. TODO: add words on
```
Comment (dims) [inline liaisons.md:97]: add a `-` and `B`

File: generator/liaisons.tmpl [hunk 8]
```diff
@@ -2,17 +2,24 @@
 
 # Liaisons
 
-Each community group in Kubernetes is assigned a Steering Committee
-liaison. Liaisons act as a point of contact from steering, engage with
-their respective community groups to ensure they are healthy and
-facilitate communication for [annual reports](committee-steering/governance/annual-reports.md).
+Steering Committee members are elected representatives of the body and
```
Comment (tpepper) [inline generator/liaisons.tmpl:5]: Something about "body" reads weird to me here.  Could it simply be "project"?

File: liaisons.md [hunk 9]
```diff
@@ -59,5 +66,40 @@ of SIGs, WGs and UGs.
...
+  agendas, minutes, and video recordings)
+    - Communication best practices for shared status and decision making across 
+  a WG and its sponsor SIG(s) and a SIG and its subprojects
+    - Provide consultation on how to improve operational health
+  - Gap Analysis
+    - Where is the group now and where do they want/need to be?
+- Act as conduit for incoming cross-cutting deliverables requirements, such as 
+ones relative to test, infra, org, docs, etc.
+- Check that sustainability is a priority. 
+  - As contributors and maintainers come and go, what is the group doing to
```
Comment (tpepper) [inline liaisons.md:90]: Super minor nit: split this bullet of three question prompts into three bullets.

**Comment by tpepper** [comment]
Aside from a couple minor formating/wording notes, lgtm!

**Comment by dims** [comment]
/lgtm
/hold cancel

we can iterate if needed. let's 🚢 this!

**Pull request merged**